egs2/mls/asr2/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+kmeans_feature=wavlm_large/21  # use model_type/layer_index
+nclusters=1000
+
+src_lang=$(echo "${kmeans_feature}_full_km${nclusters}" | tr "/" "_")
+tgt_lang=en
+
+lang="all" # one of all es en fr nl it pt pl de
+data_split="full" # one of full 1h 10h
+
+train_set="mls_${lang}_train"
+valid_set="mls_${lang}_dev"
+lm_train_text=data/${lang}_lm_train.txt
+
+if [ "$lang" == "all" ]; then
+    for ln in es en fr nl it pt pl de; do
+        test_sets+="mls_${ln}_test "
+    done
+else
+    test_sets="mls_${lang}_test"
+fi
+
+asr_config=conf/train_discrete_asr_e_branchformer1.yaml
+inference_config=conf/decode_ctc0.3.yaml
+
+src_nbpe=3000
+tgt_nbpe=150
```

Comment:
maybe we should also consider using different bpe depending on the size of the data. I feel the current setting would be good for 1h, 10h, or single languages, but definitely not good for all the data

Reply:
Great point! I set the default as 10h in https://github.com/espnet/espnet/pull/5441/commits/91c2b06069b95492b9f6eee9ffb5e1cc84bdb896
I don't think we want to do the full dataset for now.